openelevationservice/server/__init__.py

```diff
@@ -10,6 +10,9 @@
 import os
 import time
 
+#from openelevationservice.server.grpc import oes_grpc_server
+#from multiprocessing import Process
```

Comment:
Delete comments if they are not necessary

Reply:
DONE